src/olympia/users/models.py

```diff
@@ -536,7 +547,8 @@ def delete(self, addon_msg='', send_delete_email=True):
         log.info(f'User ({self}: <{self.email}>) is being anonymized.')
         if send_delete_email:
             email = self._prepare_delete_email()
-        self._anonymize()
+        self.anonymize_users((self, ))
```

Comment:
Took me a bit to figure out what's going on here. So `(self, )` is a tuple that is being passed into the `users` argument of `anonymize_users`? Is that trailing comma necessary?